### DISCUSSION TITLE
Controller support

### DISCUSSION THREAD

**Discussion by Bendyg7** [HMCL-dev/HMCL-PE-CN#46] [Ideas]
Can you add the controller support to the launcher?

 It'll give the player more game experience and it's good for players who can't play with touch controls?

**Comment by NgFinn** [comment]
你好创作者。 我是来自国外的 HMCL PE 玩家，我已经翻译了您的应用程序，希望在这里将其传播给我在游戏玩家社区的朋友。 当我进入游戏时，我被告知我需要为此捐赠 2 美元，但我没有任何支付方式，如微信支付、paypal 或支付宝。 你使用不和谐吗？ 我们怎么不能联系你捐款？ 希望您能够帮助我 。 谢谢你，我和我的朋友会支持你。

**Comment by NgFinn** [comment]
Hello creators.  I am a HMCL PE player from abroad, I have translated your application with the desire to disseminate it to my friends in the gamer community here.  When I entered the game I was told that I need to donate 2$ for this but I don't have any payment methods like wechat pay , paypal or Alipay.  Do you use discord ?  How can't we contact you an donate ?  Hope you can help me .  Thank you, me and my friends will support you.

**Reply by Tungstend** [reply]
A public beta without verification will release soon, just be patient, u translate it into Vietnamese right? I have officially added your translation to the launcher, do you mind, I will credit your contribution.

**Reply by Tungstend** [reply]
Discord:
https://discord.gg/zeMNy8Wdgd

**Reply by Tungstend** [reply]
Don't discuss here, go to discord to discuss plz.
